src/init.cpp

```diff
@@ -842,6 +842,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
             return InitError(_("Payment disclosure requires -experimentalfeatures."));
         } else if (mapArgs.count("-zmergetoaddress")) {
             return InitError(_("RPC method z_mergetoaddress requires -experimentalfeatures."));
+        } else if (mapArgs.count("-printsproutr1cs")) {
```

Comment:
I suggest naming this "-savesproutr1cs", since "-printsproutr1cs" suggested to me that it was going to print it to stdout or stderr.